lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -232,6 +233,45 @@ def test_course_about_in_cart(self):
         self.assertEqual(response.status_code, 200)
         self.assertIn(in_cart_span, response.content)
 
+    def assert_enrollment_link_present(self, is_anonymous, _id=False):
+        """ Prepare ecommerce checkout data and assert if the ecommerce link is contained in the response. """
+        checkout_page = '/test_basket/'
+        sku = 'TEST123'
+        CommerceConfiguration.objects.create(
+            checkout_on_ecommerce_service=True,
+            single_course_checkout_page=checkout_page
+        )
+        course = CourseFactory.create(org='new', number='unenrolled', display_name='course')
```

Comment:
Nit: If you don't specify these kwargs, they will be automatically generated for you. Given that we don't really care what they are, this code would be cleaner if you removed these kwargs.